core-jdk8/src/main/java/org/mapstruct/Mapping.java

```diff
@@ -37,6 +37,10 @@
  * In addition, the attributes {@link #dateFormat()} and {@link #qualifiedBy()} may be used to further define the
  * mapping.
  *
+ * <p>
+ * <b>IMPORTANT NOTE:</b> the enum mapping capability is deprecated and replaced by {@link ValueMapping} it
```

Comment:
Are we raising a WARN note, too, when using `@Mapping` for enum-to-enum mapping?